tests/test_global_opts.rs

```diff
@@ -101,8 +101,8 @@ fn test_color_config() {
     // Test that --color=always is respected.
     let stdout = test_env.jj_cmd_success(&repo_path, &["--color=always", "log", "-T", "commit_id"]);
     insta::assert_snapshot!(stdout, @r###"
-    @ [1;34m230dd059e1b059aefc0da06a2e5a7dbf22362f22[0m
-    o [34m0000000000000000000000000000000000000000[0m
+    @ 230dd059e1b059aefc0da06a2e5a7dbf22362f22
+    o 0000000000000000000000000000000000000000
```

Comment:
Why did this change?

Reply:
This is clearly a problem. I wrote a test because, in my initial version, `--config 'ui.color="always"` was ignored. I tried to fix that, but now it's broken in a different way.

Reply:
Hmm, I was hoping this was just a mistake in the test because I don't see why it would have changed.

Reply:
OK, I fixed this in an ... entertaining way. I'll look at this again another day.